snuba/datasets/storages/transactions.py

```diff
@@ -98,6 +98,13 @@
     stream_loader=KafkaStreamLoader(
         processor=TransactionsMessageProcessor(), default_topic="events",
     ),
-    query_splitters=[TimeSplitQueryStrategy(timestamp_col="finish_ts")],
+    query_splitters=[
+        ColumnSplitQueryStrategy(
+            id_column="event_id",
+            project_column="project_id",
+            timestamp_column="timestamp",
```

Comment:
Why is this based on timestamp annd not finish_ts ? This receive a clickhouse query that has already been translated.

Reply:
Oops, forgot to update this when copying over, thanks